### PR TITLE
create helloworld and healthcheck

### PR DESCRIPTION
**Pull request by yanoandri**
image
image
image

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+golang.org/x/sys v0.0.0-20200323222414-85ca7c5b95cd/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
+golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6 h1:DvY3Zkh7KabQE/kfzMvYvKirSiguP9Q/veMtkYyf0o8=
+golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
+golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
+golang.org/x/text v0.3.3 h1:cokOdA+Jmi5PJGXLlLllQSgYigAEfHXJAERHVMaCc2k=
+golang.org/x/text v0.3.3/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
+golang.org/x/time v0.0.0-20201208040808-7e3f01d25324/go.mod h1:tRJNPiyCQ0inRvYxbN9jk5I+vvW/OXSQhTDSoE431IQ=
+golang.org/x/tools v0.0.0-20180917221912-90fa682c2a6e/go.mod h1:n7NCudcB/nEzxVGmLbDWY5pfWTLqBcC2KZ6jyYvM4mQ=
+gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
+gopkg.in/yaml.v2 v2.2.2/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
```
Comment (weisurya) [inline go.sum:46]: Great! It's important to include `go.sum`.

This is out of review but do you know the usage of this file?

File: .gitignore [hunk 2]
```diff
@@ -12,4 +12,4 @@
 *.out
 
 # Dependency directories (remove the comment below to include it)
-# vendor/
```
Comment (weisurya) [inline .gitignore:15]: Nice! we don't need to push it into the repository

File: go.mod [hunk 3]
```diff
@@ -1,3 +1,5 @@
 module github.com/yanoandri/yano-golang-training-beginner
```
Comment (weisurya) [inline go.mod:1]: good module naming :+1:

**Review by weisurya — COMMENTED** [review]
<deleted>

**Review by weisurya — APPROVED** [review]
good job on completing your 1st Go web server! :star_struck:

File: go.sum (hunk 1 above)
Comment (yanoandri) [reply go.sum:46]: as far as i understands this is mostly like package-lock.json in javascript i assume, but it use for checksum and validating dependencies whether is direct or indirect that uses inside this project, to proof that they are not being modified

**Pull request merged**